oat-sa/extension-tao-itemqti
# Feat/aut 2192/toggle time dependant

related to https://oat-sa.atlassian.net/browse/AUT-2192

### Description

`Time dependent` item property visibility should be configurable through visibility config

### How to test 

-  kitchen : https://test-hidefeatures.playground.kitchen.it.taocloud.org/

 - local setup 

`'taoQtiItem/creator/item/property/timeDependant' => 'hide',` in `services/features` section of config registry file

`config/tao/client_lib_config_registry.conf.php`

Check item property is not present in Item authoring form
